dynamic/org.ops4j.pax.url.mvn.cfg.tmpl

```diff
@@ -92,4 +92,4 @@ org.ops4j.pax.url.mvn.defaultRepositories=file:${karaf.home}/${karaf.default.rep
 #    @noreleases : the repository does not contain any released artifacts
 #    @id=repository.id : the id for the repository, just like in the settings.xml this is optional but recommended
 #
-org.ops4j.pax.url.mvn.repositories=https://artifact-public.instana.io/artifactory/features-public@id=features@snapshots@snapshotsUpdate=always,https://artifact-public.instana.io/artifactory/shared@id=shared@snapshots@snapshotsUpdate=always
+org.ops4j.pax.url.mvn.repositories={{ getenv "INSTANA_FEATURE_MVN_URL" }},https://artifact-public.instana.io/artifactory/shared@id=shared@snapshots@snapshotsUpdate=always
```

Comment:
What happens if this is undefined? Shouldn't we have an `if` around it?

Reply:
it gets set in run.sh if undefined from env.